contrib/completion/bash/docker

```diff
@@ -104,6 +104,27 @@ __docker_pos_first_nonflag() {
 	echo $counter
 }
 
+# Transforms a multiline list of strings into a single line string
+# with the words separated by "|".
+# This is used to prepare arguments to __docker_pos_first_nonflag().
+__docker_to_alternatives() {
+	local pattern="$1"
+	pattern=${pattern//	/} # a verbatim tab!
+pattern=${pattern//
+/ }
+	pattern=${pattern// /|}
+	pattern=${pattern#|}
+	pattern=${pattern%|}
+	echo $pattern
```

Comment:
I think this could be simpler as something like: (ie, letting Bash do all the hard work more directly)
```bash
__docker_to_alternatives() {
    local parts=( $1 )
    IFS='|'
    echo "${parts[*]}"
    unset IFS
}
```